lib/images.sh

```diff
@@ -1,9 +1,12 @@
 #!/bin/bash
 
+
+export KUBERNETES_VERSION="${KUBERNETES_VERSION:-"v1.21.0"}"
```

Comment:
We do not need this here anymore, @Xenwar ?

Reply:
I have the same opinion as @furkatgofurov7 .  common.sh file has already export this varable.

Reply:
I agree. It was not there, I added it to avoid failure when it was not defined outside the file.

Reply:
removed